cli/README.md

```diff
@@ -428,12 +457,30 @@ code-push release-cordova <appName> <platform>
 [--rollout <rolloutPercentage>]
 ```
 
-The `release-cordova` command does two things in addition to running the vanilla `release` command described in the [Releasing App Updates](#releasing-app-updates) section:
+The `release-cordova` command is a Cordova-specific version of the "vanilla" [`release`](#releasing-app-updates) command, which supports all of the same parameters (e.g. `--mandatory`, `--description`), yet simplifies the process of releasing updates by performing the following additional behavior: 
+
+1. Running the `cordova prepare` command in order to generate the [update contents](#update-contents-parameter) (`www` folder) that will be released to the CodePush server.
+
+2. Inferring the [`targetBinaryVersion`](#target-binary-version-parameter) of this release by using the version name that is specified in your project's `config.xml` file.
+
+To illustrate the difference that the `release-cordova` command can make, the following is an example of how you might generate and release an update for a Cordova app using the "vanilla" `release` command:
+
+```shell
+cordova prepare ios
+code-push release MyApp ./platforms/ios/www 1.0.0
+```
+
+Achieving the equivalent behavior with the `release-cordova` command, would simply require the following command, which is generally less erorr-prone:
```

Comment:
Remove the first comma.

Although it is pretty charming as you have it, it should be "error-prone."